### PR TITLE
Update electronic-prescription-service-api.yaml

### PR DESCRIPTION
**Pull request by SachinPath**

### CODE REVIEW

File: specification/electronic-prescription-service-api.yaml [hunk 1]
```diff
@@ -55,19 +55,31 @@ info:
...
-
     ### Prescribing API
-    Prescribing flows for EPS FHIR API are now in a state of technical go-live awaiting our first of type integration.
-
-    The API is currently available for sandbox testing and integration testing, but not for production use.
-
+    There are three types of medication workflows
+    
+    #### Primary care medication workflow
+    In primary care medication workflow, the prescription is sent from a primary care service, such as a GP practise to a Pharmacy. For this workflow, the API is in [alpha](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status), meaning:
```
Comment (ri-ch) [inline specification/electronic-prescription-service-api.yaml:64]: Couple of minor typos:
_In primary care medication workflow_ > _In a primary care medication workflow_
_GP practise_ > _GP practice_

File: specification/electronic-prescription-service-api.yaml [hunk 2]
```diff
@@ -55,19 +55,31 @@ info:
...
+    - it is available for testing in the integration environment, but not for production use
+    - we might make breaking changes, but only if we cannot avoid it, and we will give advance notice
+
+    #### Secondary care medication workflow
+    In a secondary care medication workflow, the prescription is sent from a secondary care service, such as an NHS hospital to a Pharmacy. For this workflow, the API is in [beta](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#api-status), meaning:
```
Comment (ri-ch) [inline specification/electronic-prescription-service-api.yaml:69]: Lower case _p_ on _Pharmacy_

File: specification/electronic-prescription-service-api.yaml [hunk 3]
```diff
@@ -55,19 +55,31 @@ info:
...
+    - it is available in integration and production
+    - we might make breaking changes, but only if we cannot avoid it, and we will give advance notice
+    
+    #### Tertiary care medication workflow
+    This API does not support tertiary care medication workflow.
```
Comment (ri-ch) [inline specification/electronic-prescription-service-api.yaml:74]: _workflows_?

File: specification/electronic-prescription-service-api.yaml [hunk 4]
```diff
@@ -55,19 +55,31 @@ info:
...
+    
     ### Dispensing API
     Download and the mark as dispensed dispensing flows for the EPS FHIR API are in the sandbox environment only, withdraw and return are currently only referenced in this spec.
```
Comment (ri-ch) [inline specification/electronic-prescription-service-api.yaml:77]: Not part of this PR, but I think we need to look at re-writing this line, it doesn't make any sense as it is.

File: specification/electronic-prescription-service-api.yaml [hunk 5]
```diff
@@ -55,19 +55,31 @@ info:
     - [Personal Demographics Service - FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) - use this to obtain patient data
 
     ## API status and roadmap
-    The immediate EPS development priority is to support the rollout of the next-gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR-compliant prescription tracker into the next gen APIs.
+    The immediate priority of EPS development is to support the rollout of the next-gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR-compliant prescription tracker into the next gen APIs.
```
Comment (ri-ch) [inline specification/electronic-prescription-service-api.yaml:58]: I'm not sure this is an improvement. What about the _The immediate priority for EPS developement..._

File: specification/electronic-prescription-service-api.yaml (hunk 1 above)
Comment (SachinPath) [reply specification/electronic-prescription-service-api.yaml:64]: Corrected the typos

File: specification/electronic-prescription-service-api.yaml (hunk 2 above)
Comment (SachinPath) [reply specification/electronic-prescription-service-api.yaml:69]: Changed to lower case.

File: specification/electronic-prescription-service-api.yaml (hunk 3 above)
Comment (SachinPath) [reply specification/electronic-prescription-service-api.yaml:74]: Are there more than one workflow in Tertiary care?

File: specification/electronic-prescription-service-api.yaml (hunk 4 above)
Comment (SachinPath) [reply specification/electronic-prescription-service-api.yaml:77]: Reviewed it with Toby in the morning, he requested to keep it as is at the moment. To be reviewed later.

File: specification/electronic-prescription-service-api.yaml (hunk 5 above)
Comment (SachinPath) [reply specification/electronic-prescription-service-api.yaml:58]: The changed sentence is slightly easier to read.

File: specification/electronic-prescription-service-api.yaml [hunk 6]
```diff
@@ -55,25 +55,37 @@ info:
...
 
     ## Technology
     This API conforms to the [FHIR](https://digital.nhs.uk/developer/guides-and-documentation/api-technologies-at-nhs-digital#fhir) global standard for health care data exchange.
     Specifically, it is aligned with [FHIR UK Core](https://digital.nhs.uk/services/fhir-uk-core), which is built on FHIR Release 4.
 
-    You don’t need to know much about FHIR to use this API - FHIR APIs are just RESTful APIs that follow specific rules.
+    You do not need to know much about FHIR to use this API - FHIR APIs are just RESTful APIs that follow specific rules.
     In particular:
     - array names are singular, for example `line` not `lines` for address lines
     - data items that are country-specific and thus not included in the FHIR global base resources are usually wrapped in an `extension` object
```
Comment (ri-ch) [inline specification/electronic-prescription-service-api.yaml:91]: I think there is some punctuation missing from this line: 

> data items that are country-specific (and thus not allowed in the FHIR global base resources) are usually wrapped in an extension object

**Review by ri-ch — APPROVED** [review]
